configure.ac

```diff
@@ -198,6 +198,7 @@ fi
 
 MACHINE=`uname -m`
 
+NBPF_ENABLED=0
 AC_MSG_CHECKING([PF_RING nBPF])
 NBPF_HOME=${PWD}/../PF_RING/userland/nbpf
```

Comment:
A nice2have would be to let the user decide, what the path of nbpf should be.

Reply:
I add some autotool "magic": it seems to work (the user decide the path; default as before) but could you take a look, please?

Reply:
lgtm